kolibri/plugins/coach/assets/src/app.js

```diff
@@ -84,6 +86,8 @@ class CoachToolsModule extends KolibriApp {
           GroupsPage.name,
           GroupMembersPage.name,
           GroupEnrollPage.name,
+          HomePage.name,
```

Comment:
Since the related route uses `PageNames.HOME_PAGE` name, I think it'd be safer to use it here as well (it's possible they're same but as far as I can say that'd be rather a coincidence.